index.html

```diff
@@ -6316,11 +6316,18 @@ <h2><a name="13Decoder-gamma-handling">13.13 Decoder gamma
 <p>When the incoming image has unknown gamma (<a href=
 "#11gAMA"><span class="chunk">gAMA</span></a>, <a href=
 "#11sRGB"><span class="chunk">sRGB</span></a>, and <a href=
-"#11iCCP"><span class="chunk">iCCP</span></a> all absent), choose
+"#11iCCP"><span class="chunk">iCCP</span></a> all absent),
+standalone image viewers should
+choose
 a likely default gamma value, but allow the user to select a new
```

Comment:
The line wraps here are a tad weird. It doesn't really matter though so this is only a nit.